sdks/python/apache_beam/testing/synthetic_pipeline.py

```diff
@@ -83,7 +91,14 @@ def bytes(self, length):
     return self.getrandbits(length * 8).to_bytes(length, sys.byteorder)
 
 
-Generator = _Random
+def get_generator(seed: Optional[int] = None, algorithm: Optional[str] = None):
+  if algorithm is None or algorithm == 'builtin':
+    return _Random(seed)
+  else:
```

Comment:
Let's check or assert that algorithm=='lcg'?

Reply:
added check